src/wallet/wallet2.cpp

```
@@ -15402,16 +15412,22 @@ crypto::public_key wallet2::get_multisig_signing_public_key(size_t idx) const
   CHECK_AND_ASSERT_THROW_MES(m_multisig, "Wallet is not multisig");
   CHECK_AND_ASSERT_THROW_MES(idx < m_transfers.size(), "idx out of range");
-  for (const auto &k: m_transfers[idx].m_multisig_k)
+  auto &ks = m_transfers[idx].m_multisig_k;
+  for (auto it = ks.begin(); it != ks.end(); ++it)
   {
     rct::key L;
-    rct::scalarmultBase(L, k);
+    rct::scalarmultBase(L, *it);
     if (used_L.find(L) != used_L.end())
+    {
+      rct::key k = *it;
```

Clang (14.0.6)

[warning] variable name 'k' is too short, expected at least 3 characters

(readability-identifier-length)